docs/source/config.json

```diff
@@ -56,7 +56,8 @@
       "Health check": "/configuration/health-checks",
       "Apollo Studio reporting": "/configuration/apollo-telemetry",
       "Collecting metrics": "/configuration/metrics",
-      "Tracing": "/configuration/tracing"
+      "Tracing": "/configuration/tracing",
+      "Client awareness": "/managed-federation/client-awareness"
```

Comment:
In a separate PR, I plan to fold this Client awareness content directly into the GraphOS reporting article, but I'd rather get https://github.com/apollographql/router/pull/3168 merged before doing that, so this is an in-between step